### PR TITLE
add /tasks route for tasks page

### LINKED ISSUE #672: Add `/tasks` route to show tasks intead of `/`

### PR DESCRIPTION
**Pull request by prakashchoudhary07**
### Description:
Change make `/tasks` route to get tasks data instead of `/`

### Dev Tested:
- [x] Yes

### Images/video of the change:
image

closes #672

### CODE REVIEW

File: next.config.js
Comment (satyam73) [inline next.config.js:1]: Can we add domains into constant?

Comment (prakashchoudhary07) [reply next.config.js:1]: What domains?

Comment (satyam73) [reply next.config.js:1]: Yeah got this😅.
Haven't got the eyes there👀

**Review by satyam73 — APPROVED** [review]

**Review by vinit717 — DISMISSED** [review]

File: src/pages/tasks/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Layout from '@/components/Layout';
 import classNames from '@/styles/tasks.module.scss';
 import { TasksContent } from '@/components/tasks/TasksContent';
 
-const Index: FC = () => {
+const Task: FC = () => {
```
Comment (bhtibrewal) [inline src/pages/tasks/index.tsx:7]:
```suggestion
const Tasks: FC = () => {
```

**Review by bhtibrewal — APPROVED** [review]
LGTM

**Review by sahsisunny — APPROVED** [review]

**Review by Pratiyushkumar — APPROVED** [review]
LGTM

**Pull request merged**